.env

```diff
@@ -0,0 +1 @@
+export MONGO_URL=mongodb uri
```

Comment:
This looks interesting, the variable says "MONGO_URL" but the value says "mongodb uri". Why both are using different semantics?

What do you think about have adding some example?
```
export MONGODB_URI=mongodb://localhost/survey-test

# or (better for me)

export MONGODB_URI=mongodb://localhost/test
```